examples/common/pigweed/protos/fabric_admin_service.proto

```diff
@@ -26,5 +26,6 @@ message OperationStatus {
 
 service FabricAdmin {
   rpc OpenCommissioningWindow(DeviceCommissioningWindowInfo) returns (OperationStatus){}
+  rpc ReverseCommissionBridge(DeviceCommissioningWindowInfo) returns (pw.protobuf.Empty){}  
```

Comment:
Why are we renaming this here instead of calling it what it really is CommissionNode. We are not validating the device type during commissioning process and we are blindly just commissioning this device (as we should as per the spec test of Commissioner Control Cluster)

Reply:
CommissionNode is sent from the E1 to E2 to notify the E2 reverse commission E1 bridge. They are not sent in the same direction.

Reply:
Per our offline discussion, we identified that the term "Reverse" caused confusion because it has different meanings depending on whether it's from the perspective of the sender or the executor of the CommissionNode command.

To resolve this, we agreed to use "SendCommissionNode" for the side sending the CommissionNode command and "CommissionNode" for the side executing the command.

Reply:
Have you created a github issue so we can follow up with Chris on to make sure we got the nomenclature correct

Reply:
Which part is still not correct? we've already removed the confusing parts related to 'Reverse' from the code. We can double-check with Chris in the next sync meeting to clarify anything further.

Reply:
I think it is all correct, but we should double check with Chris that it is all correct

Reply:
https://github.com/project-chip/connectedhomeip/issues/35042